src/server.h

```diff
@@ -117,6 +117,14 @@ struct ServerPlayingSound
 	std::unordered_set<session_t> clients; // peer ids
 };
 
+struct MinimapMode {
+	u16 type = 0;
```

Comment:
Shouldn't this be `MinimapType`? I'd prefer to see this converted to `u16` in the (de)serialization code.

Reply:
I have to put MinimapType definition in a file included in both client and server.

Shoud I create a minimap.h in src (same logic as hud.h) with only this definition or do you think there is a better place for it?

Reply:
IMO putting it into `src/hud.h` would be good enough. I don't think a new header is required - and it's a HUD element after all.